RickAndMorty-iOS/Views/Locations/Cells/LocationRowCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LocationRowCell.swift
+//  RickAndMorty-iOS
+//
+//  Created by Calvin Pak on 2023-03-13.
+//
+
+import UIKit
+
+class LocationRowCell: RowCell {
+
+    static let identifier = "LocationRowCell"
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+        addStyles()
+        setupConstraints()
+    }
+
+    func addStyles() {
+        self.contentView.backgroundColor = UIColor(red: 0.99, green: 0.89, blue: 0.54, alpha: 0.1)
```

Comment:
Colours used across the app. We need to define either enum either extension for this. Each class does the same, declaring same colors